[2025-02-11T17:08:16.084+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-11T17:08:16.118+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_INETUM_ETL.preprocess_data manual__2025-02-11T17:08:00.921612+00:00 [queued]>
[2025-02-11T17:08:16.134+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_INETUM_ETL.preprocess_data manual__2025-02-11T17:08:00.921612+00:00 [queued]>
[2025-02-11T17:08:16.136+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-02-11T17:08:16.163+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): preprocess_data> on 2025-02-11 17:08:00.921612+00:00
[2025-02-11T17:08:16.176+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2764) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-11T17:08:16.180+0000] {standard_task_runner.py:63} INFO - Started process 2766 to run task
[2025-02-11T17:08:16.180+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'DAG_INETUM_ETL', 'preprocess_data', 'manual__2025-02-11T17:08:00.921612+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmpz9aj9prt']
[2025-02-11T17:08:16.182+0000] {standard_task_runner.py:91} INFO - Job 21: Subtask preprocess_data
[2025-02-11T17:08:16.286+0000] {task_command.py:426} INFO - Running <TaskInstance: DAG_INETUM_ETL.preprocess_data manual__2025-02-11T17:08:00.921612+00:00 [running]> on host 9a237c96c334
[2025-02-11T17:08:16.500+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Andres Urrea' AIRFLOW_CTX_DAG_ID='DAG_INETUM_ETL' AIRFLOW_CTX_TASK_ID='preprocess_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-11T17:08:00.921612+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-11T17:08:00.921612+00:00'
[2025-02-11T17:08:16.503+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-11T17:08:16.597+0000] {dag_etl.py:66} INFO - Proceso de preprocess_data realizado con exito, se extrajeron 500 registros unicos
[2025-02-11T17:08:16.598+0000] {dag_etl.py:67} INFO - El archivo temporal se guardo en la ruta /tmp/archivo.json
[2025-02-11T17:08:16.600+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-11T17:08:16.601+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-11T17:08:16.621+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=DAG_INETUM_ETL, task_id=preprocess_data, execution_date=20250211T170800, start_date=20250211T170816, end_date=20250211T170816
[2025-02-11T17:08:16.677+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-11T17:08:16.727+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-11T17:08:16.730+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
